x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/azureopenai/embeddings/AzureOpenAiEmbeddingsServiceSettings.java

```diff
@@ -172,6 +197,11 @@ public Integer maxInputTokens() {
         return maxInputTokens;
     }
 
+    @Override
+    public SimilarityMeasure similarity() {
+        return SimilarityMeasure.DOT_PRODUCT;
```

Comment:
Just a reminder this should be whatever the user specified, let's not default it here. When the inference endpoint is created, if the user didn't specify a similarity field, we'll set the default when the `updateModelWithEmbeddingDetails` method is called.